assets/scss/common/_page-header.scss

```diff
@@ -164,24 +164,27 @@
 .page-header-btn {
   display: block;
   width: 100%;
-  margin-top: 40px;
-
-  @include medium-and-up {
-    margin-top: 24px;
+  margin-top: 40px;
+
+  @include medium-and-up {
+    margin-top: 24px;
   }
 
   @include large-and-up {
     margin-top: 24px;
-  }
-
-  @include x-large-and-up {
-    margin-top: 36px;
+  }
+
+  @include x-large-and-up {
+    margin-top: 36px;
   }
 }
 
 .page-header-content {
   display: none;
-
+  hyphens: none;
+  font-family: "Lora", serif;

```

Comment:
Why don't you use `$lora` here?

Reply:
Ohh yeah good catch, I'll make the change.